### PR TITLE
please make further changes to readme

### PR DESCRIPTION
**Pull request by TheCodedJourney**

### CODE REVIEW

**Review by valentinbiz — APPROVED** [review]
Nice one, good to go for now

File: README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+2. Navigate to the bench upload page and follow the on screen steps to get started.
+3. Recieve confirmation of your changes.
+4. Book a session at your newly uploaded bench by following the steps in the section above
+
+## Contribution guidelines
+
+If you would like to contribute to the project, please follow the below guidelines:
+
+1. Fork the repository
+1. Create a branch for your feature
```
Comment (hsnnkb-dev) [inline README.md:118]: numbering off

File: README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+```
+npm install
+```
+
+## Backend Configuration
+
+BenchIt uses Firebase as the backend database. To configure the backend, follow these steps:
+
+1. Create a Firebase project on the Firebase website.
+1. From the Firebase Console, select the project that you just created.
```
Comment (hsnnkb-dev) [inline README.md:82]: numbering off

File: README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+- Emulators: Virtual devices used for testing and debugging the app during development. The team used emulators to simulate different devices and platforms, ensuring the app works as expected on a wide range of devices.
+- Cypress: A JavaScript end-to-end testing framework, which was used to test the app's functionality and catch bugs before release.
+
+With these technologies, the team was able to build a high-quality, user-friendly app that delivers a seamless experience on multiple platforms.
+
+## Prerequisites
+
+Before getting started with the app, you'll need to install the following tools and technologies on your machine:
+
+- [Node.js](https://nodejs.org/en/download/)
```
Comment (hsnnkb-dev) [inline README.md:51]: version numbers?

Comment (hsnnkb-dev) [inline README.md:51]: version numbers

**Review by hsnnkb-dev — APPROVED** [review]
Great work again Tom, added a few comments, maybe more detail into version numbers, maybe a few emojis to add a bit more character

**Pull request merged**